src/Reanimated.res

```diff
@@ -1,9 +1,340 @@
+  external bounceInRight: t = "BounceInRight"
+  @module("react-native-reanimated")
+  external bounceInLeft: t = "BounceInLeft"
+  @module("react-native-reanimated")
+  external bounceInUp: t = "BounceInUp"
+  @module("react-native-reanimated")
+  external bounceInDown: t = "BounceInDown"
+  @module("react-native-reanimated")
+  external flipInYRight: t = "FlipInYRight"
+  @module("react-native-reanimated")
+  external flipInYLeft: t = "FlipInYLeft"
+  @module("react-native-reanimated")
+  external flipInXUp: t = "FlipInXUp"
+  @module("react-native-reanimated")
+  external flipInXDown: t = "FlipInXDown"
+  @module("react-native-reanimated")
+  external flipInEasyX: t = "FlipInEasyX"
+  @module("react-native-reanimated")
+  external flipInEasyY: t = "FlipInEasyY"
+  @module("react-native-reanimated")
+  external stretchInX: t = "StretchInX"
+  @module("react-native-reanimated")
+  external stretchInY: t = "StretchInY"
+  @module("react-native-reanimated")
+  external zoomIn: t = "ZoomIn"
+  @module("react-native-reanimated")
+  external zoomInRotate: t = "ZoomInRotate"
+  @module("react-native-reanimated")
+  external zoomInRight: t = "ZoomInRight"
+  @module("react-native-reanimated")
+  external zoomInLeft: t = "ZoomInLeft"
+  @module("react-native-reanimated")
+  external zoomInUp: t = "ZoomInUp"
+  @module("react-native-reanimated")
+  external zoomInDown: t = "ZoomInDown"
+  @module("react-native-reanimated")
+  external zoomInEasyUp: t = "ZoomInEasyUp"
+  @module("react-native-reanimated")
+  external zoomInEasyDown: t = "ZoomInEasyDown"
+  @module("react-native-reanimated")
+  external slideInRight: t = "SlideInRight"
+  @module("react-native-reanimated")
+  external slideInLeft: t = "SlideInLeft"
+  @module("react-native-reanimated")
+  external slideInUp: t = "SlideInUp"
+  @module("react-native-reanimated")
+  external slideInDown: t = "SlideInDown"
+  @module("react-native-reanimated")
+  external lightSpeedInRight: t = "LightSpeedInRight"
+  @module("react-native-reanimated")
+  external lightSpeedInLeft: t = "LightSpeedInLeft"
+  @module("react-native-reanimated")
+  external pinwheelIn: t = "PinwheelIn"
+  @module("react-native-reanimated")
+  external rollInLeft: t = "RollInLeft"
+  @module("react-native-reanimated")
+  external rollInRight: t = "RollInRight"
+  @module("react-native-reanimated")
+  external rotateInDownLeft: t = "RotateInDownLeft"
+  @module("react-native-reanimated")
+  external rotateInDownRight: t = "RotateInDownRight"
+  @module("react-native-reanimated")
+  external rotateInUpLeft: t = "RotateInUpLeft"
+  @module("react-native-reanimated")
+  external rotateInUpRight: t = "RotateInUpRight"
+}
+include LayoutAnimation
+
+module KeyFrame = {
+  type keyframeConfig = {
+    from?: reanimatedStyle,
+    to?: reanimatedStyle,
+    // This looks a little bit stupid, but this has better developer experience and type inference than using a dict/object.
+    \"0"?: reanimatedStyle,
+    \"1"?: reanimatedStyle,
```

Comment:
Could we use `@as` decorator here and have be a bit easier to use?
For example:

```rescript
type keyframeConfig = {
  @as("0") _0: float,
  @as("1") _1: float,
}

// and then when using

let keyframeConfig = { _0: style, _1: style }
```

In my opinion, it would be easier to write an underscore (`_`), instead of `\""` 😃

Reply:
Good ideia! I will do this

Reply:
Fixed